helm-charts/azure-api-management-gateway/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 appVersion: "0.1"
 description: A Helm chart to deploy an Azure API Management Gateway on Kubernetes
 name: azure-api-management-gateway
-version: 0.1.0
+version: 0.1.1
```

Comment:
Can you rever this one please, we'll do a separate bump for this if ok?

Reply:
Done